### ISSUE TITLE
 Empty String for variables

### ISSUE THREAD

**Issue by Ngob** [webonyx/graphql-php#156]
Hello,

I wonder why an empty string for the variable's field (in the JSON request) is not handled as a `null` value?
It would allow the `StandardServer` to be used with the chrome's plugin "Graphiql Feen".

For example, the JSON request made by "Graphiql Feen" is 
```json
{"query":"query {\n  billPerYear {\n    id\n  }\n}\n","variables":""}
```
And the response is:
```json
{"errors":[{"message":"GraphQL Request parameter \u0022variables\u0022 must be object or JSON string parsed to object, but got (empty string)","category":"request"}]}
```
Did i missunderstood something?

I can make the change if needed.

Thanks.

**Comment by timur560** [comment]
I've fixed it just by additional check like:
```php
if (empty($variables)) {
	$variables = null;
}
```
But yes, it would be great to fix this in library code.

**Comment by Ngob** [comment]
@timur560 I am using `StandardServer` with the PSR-7 method, so I fixed it this way
```

      if (strtoupper($request->getMethod()) == "GET") {
        $params = $request->getQueryParams();
        $params["variables"] = empty($params["variables"]) ? null : $params["variables"];
        $request = $request->withQueryParams($params);
      } else {
         $params = $request->getParsedBody();
         $params["variables"] = empty($params["variables"]) ? null : $params["variables"];
         $request = $request->withParsedBody($params);
      }
```
But it feels so hacky :(
By the way, thanks for your help :)

**Comment by vladar** [comment]
Yeah, it makes sense. Fixed in master, will release 0.10.2 with a fix in couple minutes
